examples/gradle/dokka-customFormat-example/build.gradle.kts

```diff
@@ -31,6 +31,5 @@ tasks.dokkaHtml {
 }
 
 dependencies {
-    implementation(kotlin("stdlib"))
```

Comment:
> You no longer need to declare a dependency on the `stdlib` library in any Kotlin Gradle project, including a multiplatform one. The dependency is added by default.

https://kotlinlang.org/docs/whatsnew14.html#dependency-on-the-standard-library-added-by-default
 
Therefore I removed it from all subprojects and examples.